docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+**Who Should Use This Guide?**
+
+This guide is ideal for beginners new to Slurm and HPC, researchers intending to use Slurm-based clusters for their computation tasks, system administrators managing HPC environments, and even seasoned HPC users looking to brush up on their knowledge. It progresses from fundamental to advanced topics, making it a valuable resource for a broad audience.
+
+## Slurm: Basic Concepts
+Before we delve into using Slurm, it's essential to grasp some fundamental concepts related to its operation.
+
+### Nodes
+In the context of Slurm, a 'node' refers to a server within the HPC cluster. Each node possesses its resources, such as CPUs, memory, storage, and potentially GPUs. Slurm manages these nodes and allocates resources to the tasks.
+
+(slurm-partitions)=
+### Partition(s)
+A 'partition' is a grouping of nodes. You can think of partitions as virtual clusters within your HPC system. They allow administrators to segregate the compute environment based on factors like job sizes, hardware type, or resource allocation policies.
+
+:::{seealso}
+Our {ref}`partition-names` documentation.
+:::
+
+### Account information
+Some Discovery users have more than one Discovery group account associated with their usernames. For example, a student might be in a class using Discovery and a student club using Discovery for a club project. In this case, the student would have two group accounts associated with their username.
+
+When running a job with either `srun` or `sbatch`, if you have more than one account associated with your username, we recommend you use the `--account=` flag and specify the account that corresponds to the respective project. In the example with a student associated with a class and a student club, if the student is on Discovery submitting a job for a project for their class, set the `account=` flag to the name of the class account. If the student is working on a project for the club, set the `account=` flag to the name of the student club account.
+
+To find out what account(s) your username is associated with, use the following command:
+
+:::{code} bash
+sacctmgr show associations user=<yourusername>
+:::
+
+After you have determined what accounts your username is associated with, if you have more than one account association, you can use the `account=` flag with your usual `srun` or `sbatch` commands.
+
+For example, if you are associated with an account named `dataclub` and an account named `info7500`, and you’re currently doing work that should be associated with the `dataclub` account, in your `srun` command, you can add the `--account=dataclub` flag to specify that account:
```

Comment:
This feel redundant with the paragraph starting at line 42. I would shorted the paragraph at line 42 and keep the specifics here.

Reply:
This was removed.